test/test_list.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# set -uo pipefail;
+
 declare -a errors
 
 function error_and_proceed() {
```

Comment:
I think we can move this function to `helpers` instead of repeating it on every test file.